### PR TITLE
intial commit

### PR DESCRIPTION
**Pull request by Subhransupanda2000**
Screenshot (645)

### CODE REVIEW

File: src/main/java/com/example/bookstoreapp/models/MediaType.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.example.bookstoreapp.models;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum MediaType {
+    IMAGE(0), VIDEO(1);
```
Comment (rkpatra201) [inline src/main/java/com/example/bookstoreapp/models/MediaType.java:7]: I think we are not using these weights. We should use these to store in db.

File: src/main/java/com/example/bookstoreapp/entities/MediaEntity.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@Table(name = "media")
+@Entity
+@NoArgsConstructor
+public class MediaEntity {
+    @Id
+   private Long id;
+  private   MediaType mediaType; // IMG, VIDEO
```
Comment (rkpatra201) [inline src/main/java/com/example/bookstoreapp/entities/MediaEntity.java:17]: Flyway script says the database column is tiny int but here it is declared as media type and this is inconsistent.

**Review by rkpatra201 — COMMENTED** [review]
Reviewed

File: src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MediaServiceImplTest {
```
Comment (rkpatra201) [inline src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java:17]: Your class file name is MediaServiceImpl (src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java) but the class name defined at line 17 here I see MediaServiceImplTest.

I think both should named as MediaServiceImplTest.

**Review by rkpatra201 — COMMENTED** [review]
Reviewed.

**Comment by rkpatra201** [comment]
@Subhransupanda2000 nice and clean work in this PR.

File: src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java (hunk 3 above)
Comment (Subhransupanda2000) [reply src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java:17]: ok

**Pull request closed without merging**